src/status_im/chat/models/images.cljs

```diff
@@ -37,31 +39,27 @@
     (.-localIdentifier result)
     (.-path result)))
 
-(defn android-save-image-to-gallery [base64-uri]
-  (react/image-get-size
-   base64-uri
-   (fn [width height]
-     (image-processing/resize
-      base64-uri
-      width
-      height
-      100
-      (fn [^js resized-image]
-        (let [path (.-path resized-image)
-              path (if (string/starts-with? path "file") path (str "file://" path))]
-          (.save CameraRoll path)))
-      #(log/error "could not resize image" %)))))
+(def temp-image-url (str (fs/cache-dir) "/StatusIm_Image.jpeg"))
```

Comment:
could you please elaborate, why this needed ? what's this image ?

Reply:
CameraRoll is expecting a file URL so we have to first download the image locally to cache. Before this was done by saving the base64-encoded image using image-processing/resize, but this didn't work with HTTP(s) links. I added a way to download the image as a binary and unified the code/logic used for saving to gallery and to share image and also made it the same for iOS and Android